Sources/Subs-Db-postgresql.php

```diff
@@ -413,19 +415,9 @@ function smf_db_query($identifier, $db_string, $db_values = array(), $connection
 	// First, we clean strings out of the query, reduce whitespace, lowercase, and trim - so we can check it over.
 	if (empty($modSettings['disableQueryCheck']))
 	{
-		$clean = preg_split('/(?<![\'\\\\])\'(?![\'])/', $db_string);
-
-		for ($i = 0; $i < count($clean); $i++)
-		{
-			if ($i % 2 === 1)
-				$clean[$i] = ' %s ';
-		}
-
-		$clean = trim(strtolower(preg_replace(
-			$allowed_comments_from,
-			$allowed_comments_to,
-			implode('', $clean)
-		)));
+		// Clear out escaped single quotes first, to make it simpler to ID & remove string literals
+		$clean = str_replace('\'\'', '', $db_string);
```

Comment:
```suggestion
		$clean = str_replace('\'\'', ' ', $db_string);
```

Here is one more important recommendation. By replacing the escaped quotation mark with a space rather than nothing, we can avoid accidentally creating strings that contain unexpected syntax structures that were not present in the original.